package.json

```diff
@@ -39,57 +39,49 @@
-    "@babel/plugin-proposal-throw-expressions": "^7.1.2",
-    "@babel/plugin-syntax-dynamic-import": "^7.1.2",
-    "@babel/plugin-syntax-import-meta": "^7.1.2",
-    "@babel/plugin-transform-runtime": "^7.1.2",
-    "@babel/preset-env": "^7.1.2",
-    "@babel/preset-flow": "^7.0.0",
-    "@babel/runtime": "^7.1.2",
-    "@integreat-app/eslint-config-babel": "^0.3.4",
-    "@integreat-app/eslint-config-flow": "^0.3.2",
-    "@integreat-app/eslint-config-jest": "^0.2.1",
-    "@integreat-app/eslint-config-node": "^0.3.4",
-    "babel-eslint": "^10.0.1",
+    "@babel/cli": "^7.8.3",
+    "@babel/core": "^7.8.3",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-do-expressions": "^7.8.3",
+    "@babel/plugin-proposal-export-default-from": "^7.8.3",
+    "@babel/plugin-proposal-export-namespace-from": "^7.8.3",
+    "@babel/plugin-proposal-json-strings": "^7.8.3",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.8.3",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.8.3",
+    "@babel/plugin-proposal-numeric-separator": "^7.8.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.8.3",
+    "@babel/plugin-proposal-pipeline-operator": "^7.8.3",
+    "@babel/plugin-proposal-throw-expressions": "^7.8.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-syntax-import-meta": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/preset-env": "^7.9.5",
+    "@babel/preset-flow": "^7.8.3",
+    "@babel/runtime": "^7.9.2",
+    "@integreat-app/eslint-config-babel": "^0.6.0",
+    "@integreat-app/eslint-config-flow": "^0.6.0",
+    "@integreat-app/eslint-config-jest": "^0.6.0",
+    "@integreat-app/eslint-config-node": "^0.6.0",
+    "babel-eslint": "^10.1.0",
     "babel-jest": "^24.0.0",
-    "babel-loader": "^8.0.5",
+    "babel-loader": "^8.1.0",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "es-check": "^5.0.0",
-    "eslint": "^5.12.1",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-flowtype": "^3.2.1",
-    "eslint-plugin-import": "^2.15.0",
-    "eslint-plugin-jest": "^22.1.3",
-    "eslint-plugin-node": "^8.0.1",
-    "eslint-plugin-prefer-object-spread": "^1.2.1",
-    "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0",
-    "flow-bin": "^0.91.0",
-    "flow-copy-source": "^2.0.2",
+    "es-check": "^5.1.0",
+    "eslint": "^6.8.0",
+    "flow-bin": "~0.117.0",
+    "flow-copy-source": "^2.0.9",
     "flow-typed": "^2.5.1",
     "jest": "^24.0.0",
-    "jest-fetch-mock": "^2.1.0",
-    "jest-junit": "^6.1.0",
+    "jest-fetch-mock": "^3.0.3",
+    "jest-junit": "^10.0.0",
     "shelljs": "^0.8.3",
     "uglifyjs-webpack-plugin": "^2.1.2",
-    "webpack": "^4.29.0",
-    "webpack-cli": "^3.2.1",
+    "webpack": "^4.42.1",
+    "webpack-cli": "^3.3.11",
     "webpack-node-externals": "^1.7.2"
   },
   "dependencies": {
-    "entities": "^1.1",
+    "entities": "^2.0.0",
```

Comment:
Do we have tests which use this library? I can remember that version 2 caused problems I think.

Reply:
Yes, we only use it in the events endpoint to decode excerpts. We have a test with a string with special chars.

Reply:
Do you recall what caused problems and what problems?

Reply:
@maxammann ping

Reply:
No I don't remember, feel free to merge :)